openedx/core/djangoapps/user_authn/templates/user_authn/edx_ace/passwordresetsuccess/email/body.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'ace_common/edx_ace/common/base_body.html' %}
+
+{% load i18n %}
+{% load static %}
+{% block content %}
+<table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
+    <tr>
+        <td>
+            <h1>
+                {% filter force_escape %}
+                {% blocktrans %}Your {{ platform_name }} password has changed{% endblocktrans %}
+                {% endfilter %}
+            </h1>
+            <p style="color: rgba(0,0,0,.75);">
+                {% filter force_escape %}
+                {% blocktrans %}Hello {{ name }},{% endblocktrans %}
+                {% endfilter %}
+            </p>
+            <p style="color: rgba(0,0,0,.75);">
+                {% filter force_escape %}
+                {% blocktrans %}This is to confirm that you have successfully changed your password associated with {{ platform_name }}. Please sign-in to your account.{% endblocktrans %}
+                {% endfilter %}
+                <br />
+            </p>
+
+                {% filter force_escape %}
+                {% blocktrans asvar course_cta_text %}Sign in{% endblocktrans %}
+                {% endfilter %}
+                {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text course_cta_url=login_link %}
+
+            <p>
+                {% trans "Best regards," as tmsg %}{{ tmsg | force_escape }}
+                <br/>
+                {% filter force_escape %}
+                {% blocktrans %}The {{ platform_name }} Team {% endblocktrans %}
+                {% endfilter %}
```

Comment:
Such messages are usually not added in the post-password reset emails unless it is a requirement.

Reply:
Removed this